includes/Modules/AdSense.php

```diff
@@ -797,6 +801,25 @@ private function register_tag() {
 				$tag->register();
 			}
 		}
+
+		if ( Feature_Flags::enabled( 'adBlockerDetection' ) && ! $this->context->is_amp() ) {
+			$ad_blocking_recovery_web_tag = new Ad_Blocking_Recovery_Web_Tag( $settings['accountID'], self::MODULE_SLUG );
+			$this->web_tag->set_ad_blocking_recovery_tag( $this->ad_blocking_recovery_tag );
+			$this->web_tag->set_use_error_snippet( $this->get_settings()->get()['useAdBlockerDetectionErrorSnippet'] );
+
+			if ( $ad_blocking_recovery_web_tag->is_tag_blocked() ) {
```

Comment:
This call will return the same result as the AdSense tags since the class uses the same module slug which means this call would be redundant. Either we should return early on the first call, or remove this condition here to render this regardless.

Reply:
It makes sense to block the ABR tag if the adsense web tag was blocked programmatically, what do you think?